### PR TITLE
Avoid "Unknown severity" message for fatal

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@davidrohr @shahor02 I think this is the culprit...

File: logger/Logger.cxx [hunk 1]
```diff
@@ -66,6 +66,7 @@ const unordered_map<string, Severity> Logger::fSeverityMap =
 {
     { "nolog",     Severity::nolog     },
     { "NOLOG",     Severity::nolog     },
+    { "fatal",     Severity::fatal     },
```
Comment (rbx) [inline logger/Logger.cxx:69]:
```suggestion
    { "fatal",     Severity::fatal     },
    { "FATAL",     Severity::fatal     },
```

Comment (rbx) [reply logger/Logger.cxx:69]: Thanks for the PR. It is a bug. And Fatal is one of the "old" severities, where uppercase was relevant/used, so I'd add it too.

**Review by rbx — APPROVED** [review]

**Comment by rbx** [comment]
Released with v1.11.1

**Pull request merged**